packages/gatsby/src/redux/reducers/html.ts

```diff
@@ -87,20 +87,7 @@ export function htmlReducer(
     }
 
     case `PAGE_QUERY_RUN`: {
-      if (action.payload.isPage) {
-        const htmlFile = state.trackedHtmlFiles.get(action.payload.path)
-        if (!htmlFile) {
-          // invariant
-          throw new Error(
-            `[html reducer] I received event that query for a page finished running, but I'm not aware of the page it ran for (?)`
-          )
-        }
-
-        if (htmlFile.pageQueryHash !== action.payload.resultHash) {
-          htmlFile.pageQueryHash = action.payload.resultHash
-          htmlFile.dirty |= FLAG_DIRTY_PAGE_QUERY_CHANGED
-        }
-      } else {
+      if (!action.payload.isPage) {
```

Comment:
Was surprised at first when I saw that this action only handles static query case. Maybe we can have a comment here mentioning that the page query case is handled in `ADD_PAGE_DATA_STATS`?